src/vcpkg/base/files.cpp

```diff
@@ -3626,11 +3626,8 @@ namespace vcpkg
         }
     };
 
-    Filesystem& get_real_filesystem()
-    {
-        static RealFilesystem real_fs;
-        return real_fs;
-    }
+    static RealFilesystem real_filesystem_instance;
```

Comment:
I would like to make this constexpr but that would force making all the member functions const and I'm not sure people will go for that...